recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2018.05.22" %}
+{% set version = "2018.05.24" %}
```

Comment:
Do we want to do the version bump here or were there other things we need to get in first?

cc @conda-forge/core

Reply:
We can always do another release.  I don't think it is a big issue